pages/linux/hcitool.md

```diff
@@ -0,0 +1,36 @@
+# hcitool
+
+> Monitor & Configure Bluetooth connections.
+> Send special commands to Bluetooth devices.
+
+- Scan for remote devices:
+
+`hcitool scan`
+
+- Output name of the device with its mac address:
+
+`hcitool name {{bdaddr}}`
+
+- Fetch info of a remote bluetooth device:
+
+`hcitool info {{bdaddr}}`
+
+- Check link quality:
+
+`hcitool lq {{bdaddr}}`
+
+- Display transmit power level:
+
+`hcitool tpl {{bdaddr}} {{type}}`
+
+- Check link policy:
+
+`hcitool lp`
+
+- Request authentication:
+
+`hcitool auth {{bdaddr}}`
+
+- Display local devices:
```

Comment:
What are local devices exactly? Are those the Bluetooth adapters?

Reply:
Local devices  are basically devices with Bluetooth card(s) for transmitting/receiving data.